ugh/.gitignore

```diff
@@ -1,2 +1,3 @@
 /build
 /dist
+/bin
```

Comment:
Ant and ant tools are not creating directory called `bin`. Maybe your build environment must be adjusted or add `/bin` to your global git ignore list. IMHO: There is no need to add more directories to the ignore list

Reply:
Well, you're right, it has nothing to do with Ant. Eclipse IDE did this, according to the Eclipse-configuration that is already included in the Repository Branch 2.x at `ugh/.project`